.github/workflows/postman-integration-testing.yml

```diff
@@ -0,0 +1,74 @@
+name: postman-integration-testing
+
+on:
+  workflow_call:
+    inputs:
+      repository_name:
+        description: "Repository Name <owner/repo> where postman collection is located"
+        type: string
+        required: true
+      repository_branch:
+        description: "Repository Branch where postman collection will be used"
+        type: string
+        required: true
+      postman_folder:
+        description: "Postman folder that the integration test will execute"
+        type: string
+        required: true
+      postman_collection_filepath: 
+        description: "Postman Collection Filename relative to root of the repository"
+        type: string
+        required: true
+      postman_environment_filepath: 
+        description: "Postman Environment Filename relative to root of the repository"
+        type: string
+        required: true
+    secrets:
+        repository_token:
+          description: "github token for private repository"
+          required: true
+        slack_webhook_url:
+          description: "Slack Webhook URL"
+          required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
let's upgrade this ?

Reply:
@niroz89 , looks like using this might be the optimal approach. Postman v10 is currently supporting it by generating its own github actions

https://learning.postman.com/docs/integrations/available-integrations/ci-integrations/github-actions/#configuring-the-postman-cli-for-github-actions

Reply:
hmm.. how does the generated workflow file look like ? still we need to keep it in a reusable workflow right ?

Reply:
Upgraded to 18.x . They are using postman cloud to execute the integration test which has different approach to our reusable workflow